owncloudData/src/main/java/com/owncloud/android/data/sharing/shares/datasources/LocalShareDataSource.kt

```diff
@@ -20,27 +20,27 @@
 package com.owncloud.android.data.sharing.shares.datasources
 
 import androidx.lifecycle.LiveData
-import com.owncloud.android.lib.resources.shares.ShareType
-import com.owncloud.android.data.sharing.shares.db.OCShareEntity
+import com.owncloud.android.domain.sharing.shares.model.OCShare
+import com.owncloud.android.domain.sharing.shares.model.ShareType
 
 interface LocalShareDataSource {
     fun getSharesAsLiveData(
         filePath: String,
         accountName: String,
         shareTypes: List<ShareType>
-    ): LiveData<List<OCShareEntity>>
+    ): LiveData<List<OCShare>?>
```

Comment:
Here we return a nullable list livedata. Maybe it would be better to return an empty list instead of a nullable list. With this, we could avoid having nullability problems in each layer. And use isEmpty instead of isNullOrEmpty to check there are no shares.

Reply:
Done